rust/src/quic/quic.rs

```diff
@@ -31,12 +35,14 @@ static mut ALPROTO_QUIC: AppProto = ALPROTO_UNKNOWN;
 
 const DEFAULT_DCID_LEN: usize = 16;
 const PKT_NUM_BUF_MAX_LEN: usize = 4;
+const QUIC_MAX_CRYPTO_FRAG_LEN: usize = 65535;
```

Comment:
What is a good value here ?

Reply:
If the goal is to allow a TLS client hello record, we can use the max size of that?

Reply:
🙄 

But at least TLS has the handshake in order (not multiple crypto frags out of order)